src/main/java/com/github/wvengen/maven/proguard/ProGuardMojo.java

```diff
@@ -132,6 +133,19 @@
 	 */
 	private boolean putLibraryJarsInTempDir;
 
+	/**
+	 * Create a temporary configuration file
```

Comment:
Needs a bit more explanation for this feature.
For example: "Use this parameter if your command line arguments become too long and execution fails. If this parameter is true, the arguments are written to a file and the Proguard process reads them from the file instead of the command itself.

Reply:
Done. Also updated the maven documentation.